.github/actions/install/action.yml

```diff
@@ -0,0 +1,15 @@
+name: Setup python environenment
```

Comment:
```suggestion
name: Setup python environment
```
Could you also name the directory so it matches?

Reply:
:+1: